src/internal_events/loki.rs

```diff
@@ -1,13 +1,22 @@
+use super::prelude::{error_stage, error_type};
 use metrics::counter;
 use vector_core::internal_event::InternalEvent;
 
 #[derive(Debug)]
-pub struct LokiEventUnlabeled;
+pub struct LokiEventUnlabeledError;
 
-impl InternalEvent for LokiEventUnlabeled {
+impl InternalEvent for LokiEventUnlabeledError {
     fn emit_metrics(&self) {
-        counter!("processing_errors_total", 1,
-                "error_type" => "unlabeled_event");
+        counter!(
+            "component_errors_total", 1,
+            "error_code" => "unlabeled",
```

Comment:
I think I actually like `unlabeled_event` here better. Users might otherwise interpret `unlabeled` to mean the error code is unknown.